user/incus/template.py

```diff
@@ -0,0 +1,81 @@
+pkgname = "incus"
+pkgver = "6.4.0"
+pkgrel = 0
+build_style = "go"
+make_build_args = ["./..."]
+hostmakedepends = [
+    "automake",
+    "bash",
+    "git",
+    "go",
+    "go-md2man",
+    "libtool",
+    "pkgconf",
+]
+makedepends = [
+    "acl-devel",
+    "cowsql-devel",
+    "libatomic-chimera-devel-static",
+    "libseccomp-devel",
+    "libcap-devel",
+    "linux-headers",
+    "libuv-devel",
+    "libunwind-devel-static",
+    "lxc-devel",
+    "musl-devel-static",
+    "raft-devel",
+    "sqlite-devel",
+    "udev-devel",
+]
+depends = [
+    "acl",
```

Comment:
`acl` gets pulled in automatically as something in the package links against it, so this shouldn't be needed?